hack/dev/docker-compose.yaml

```diff
@@ -1,6 +1,24 @@
 version: '3.7'
 
 services:
+  osctl:
+    image: alpine
+    entrypoint: /bin/osctl
+    command: version
+    volumes:
+      - ../../build/osctl-linux-amd64:/bin/osctl:ro
+      - ./talosconfig:/root/.talos/config
+    networks:
+      talosbr:
+
+  kubectl:
+    image: lachlanevenson/k8s-kubectl
```

Comment:
should we just make use of the hyperkube image instead of downloading another image?

Reply:
I think this is a good idea since it would also ensure we're using the version of k8s targeted for Talos.

Reply:
Moved to hyperkube -- this version is hard-coded into the other makefile.
I hard-coded its value in docker-compose and made it overrideable.

I figure this works for now, but I'm open to other ideas.